css-sizing-4/Overview.bs

```diff
@@ -211,7 +211,7 @@ Preferred Aspect Ratios: the 'aspect-ratio' property</h3>
 			If the <<ratio>> is [=degenerate ratio|degenerate=],
 			the property instead behaves as ''aspect-ratio/auto''.
 
-		<dt><dfn>auto && <<ratio>></dfn>
+		<dt><dfn>auto || <<ratio>></dfn>
```

Comment:
Wouldn't this imply that this case also describes lone `auto` and `<ratio>`?

Reply:
Yes, please see the very first next sentence: `&&` requires them together (in any order) unconditionnally.

Reply:
I mean, lone `auto` and `<ratio>` are already defined above, so they shouldn't be included here. This case is just for the combination of them

Reply:
Oh, now I see. I assume the authors do not mind using a value definition to name a term representing a subset of the property value definition. 

Sorry, I was not reading carefully and I did not expected it, obviously.